developer_setup.md

```diff
@@ -211,6 +211,22 @@ bin/setup
 
 macOS requires platform specific Gems. Run `bundle config specific_platform true` before running `bin/setup`.
 
+The ovirt-engine-sdk gem contains C code that generated warnings with older CLANG versions.  These warnings are marked as errors in newer CLANG 15 and 16 depending on the CLANG version, architecture, and possibly OS.  If this is the case, you can mark these as warnings and not errors to allow installation:
+
+```
+gem install ovirt-engine-sdk -v4.6.0 -- --with-cflags="-Wno-error=incompatible-function-pointer-types -Wno-error=implicit-function-declaration"
```

Comment:
I'm wondering if we should remove the `-v4.6.0` here, since it could apply to future versions too and we don't want to have to keep updating this.  Besides, the application seems to use 4.4.1 anyway.